File: Easy/23PalindromeCheck.py
def isPalindrome(string):
    i = 0
    j = len(string)-1
    while i <= j:
        if string[i] == string[j]:
            i += 1
            j -= 1
        else:
            return False
    return True


print(isPalindrome('abcdcba'))

File: Easy/23PalindromeCheck.js
function isPalindrome(string) {
  let i = 0;
  let j = string.length - 1;
  while (i <= j) {
    if (string[i] == string[j]) {
      i += 1;
      j -= 1;
    } else return false;
  }
  return true;
}

console.log(isPalindrome("abcdcba"));
